src/main/java/com/gradle/QuarkusExtensionConfiguration.java

```diff
@@ -117,7 +122,8 @@ boolean isNativeBuildInContainerRequired() {
      * @return dump config file name
      */
     String getDumpConfigFileName() {
-        return String.format(".quarkus/%s-%s-%s",
+        return String.format("%s/%s-%s-%s",
```

Comment:
To avoid any issue with local file access, I would prefer to have the files in `.quarkus` folder but allow an intermediate folder structure
`.quarkus/%s/%s-%s-%s`